test/test_optim.py

```diff
@@ -744,141 +744,6 @@ def test_reduce_lr_on_plateau8(self):
                                       threshold=0.1, patience=5, cooldown=5)
         self._test_reduce_lr_on_plateau(scheduler, targets, metrics, epochs)
 
-    def test_compound_step_and_multistep_lr(self):
```

Comment:
I'm surprised that these are the only tests that failed; #14010 added a lot of things into test_optim.py

Reply:
Yeah, the other tests were just comparing against reference implementations (which still works; it's just a less interesting test.)